internal/daemon/controller/tickers.go

```diff
@@ -13,8 +14,9 @@ import (
 
 // In the future we could make this configurable
 const (
-	statusInterval      = 10 * time.Second
-	terminationInterval = 1 * time.Minute
+	workerConnectionMaintenanceInterval = 3 * time.Second
```

Comment:
Do we want a bigger time interval to ease db load/ should it at least be as long as the status interval?

Reply:
A reminder that this will change a bit with https://github.com/hashicorp/boundary/pull/2578

Reply:
Not status interval, grace and such. But just want to surface it in case it matters somewhere in here.